docs/additional_reading/react_router.md

```diff
@@ -33,3 +35,11 @@ const RouterApp = (props, location) => {
   );
 };
 ```
+
+For a fleshed out integration of react_on_rails with react-router, check out [React Webpack Rails Tutorial Code](https://github.com/shakacode/react-webpack-rails-tutorial), specifically the files:
+
+[routes](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/app/bundles/comments/routes/routes.jsx)
```

Comment:
let's use the full path `/client/something` for these

and let's use bullets for formatting